packages/adapters/bitcoin/src/connectors/SatsConnectConnector.ts

```diff
@@ -75,10 +75,17 @@ export class SatsConnectConnector extends ProviderEventEmitter implements Bitcoi
   }
 
   async connect() {
+    const currentNetwork = this.getActiveNetwork()
+    const networkName = currentNetwork
+      ? mapCaipNetworkToXverseName(currentNetwork.caipNetworkId)
```

Comment:
We can handle the case where active network `undefined` case inside the `mapCaipNetworkToXverseName`